sensor_msgs/Image image
---
string status
rasberry_perception/ObjectDetection detections
